### PR TITLE
Git guides a11y updates

### PR DESCRIPTION
**Pull request by stamat**
## Overview

Hey there, big regards from @github/brand-engineering 👋 

I'm currently updating the https://github.com/git-guides and I've stumbled on couple small accessibility violations in the guides:

1. Missing a table heading copy
2. Heading hierarchy violation

That is all! ✨

### CODE REVIEW

**Review by qiang88550 — APPROVED** [review]

**Review by tobiasahlin — APPROVED** [review]

**Review by Thapeloza — APPROVED** [review]

**Comment by GRURoot** [comment]
@qiang88550 []()_****_

File: git-guides/install-git.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Note: [`git-scm`](https://git-scm.com/download/win) is a popular and recommended
 ## Install Git on Windows through Visual Studio Code
 
 GitHub integration is provided through the [GitHub Pull Requests and Issues extension](https://marketplace.visualstudio.com/items?itemName=GitHub.vscode-pull-request-github).
-To get started with the GitHub in VS Code, you'll need to create an account and install the GitHub Pull Requests and Issues extension. 
+To get started with the GitHub in VS Code, you'll need to create an account and install the GitHub Pull Requests and Issues extension.
 Once you've installed the [GitHub Pull Requests and Issues extension](https://marketplace.visualstudio.com/items?itemName=GitHub.vscode-pull-request-github), you'll need to sign in. Follow the prompts to authenticate with GitHub and return to VS Code.
```
Comment (GRURoot) [inline git-guides/install-git.md:33]: dig TXT-RECORD-NAME +nostats +nocomments +nocmd TXThttps://github.com/enterprises/gru    {
      "meshConfig": {
        "sources": [
          {
            "name": "Commerce",
            "handler": {
              "graphql": {
                "endpoint": "https://<your_commerce_site>/graphql/"
              }
            }
          },
          {
            "name": "AEM",
            "handler": {
              "graphql": {
                "endpoint": "https://<your_AEM_site>/endpoint.json"
              }
            }
          },
          {
            "name": "CatalogService",
              "handler": {
                "graphql": {
                  "endpoint": "https://catalog-service.adobe.io/graphql/search/graphql",
                  "operationHeaders": {
                    "x-api-key": "<api_key>",
                    "Magento-Environment-Id": "<your_environment_id>",
                    "Magento-Website-Code": "base",
                    "Magento-Customer-Group": "<customer_group_value>",
                    "Magento-Store-Code": "main_website_store",
                    "Magento-Store-View-Code": "default"
                  },
                  "schemaHeaders": {
                    "x-api-key": "<api_key>"
                  }
                }
              }
            }
          ]
        }
    }

File: git-guides/install-git.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ Note: [`git-scm`](https://git-scm.com/download/win) is a popular and recommended
 ## Install Git on Windows through Visual Studio Code
 
 GitHub integration is provided through the [GitHub Pull Requests and Issues extension](https://marketplace.visualstudio.com/items?itemName=GitHub.vscode-pull-request-github).
```
Comment (GRURoot) [inline git-guides/install-git.md:31]: - Gru.github.com@[]()BeatsPillPlusAndroid.pdf

Apple-Card-Customer-Agreement.pdf

**Review by GRURoot — APPROVED** [review]

**Review by GRURoot — APPROVED** [review]

**Review by amamore480 — APPROVED** [review]
<!DOCTYPE html>
<html lang="ar">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>جمعية الخير والعطاء</title>
    <style>
        body {
            font-family: Arial, sans-serif;
            direction: rtl;
            text-align: center;
            background-color: #f8f9fa;
            margin: 0;
            padding: 0;
        }
        header {
            background-color: #28a745;
            color: white;
            padding: 15px;
            font-size: 24px;
        }
        section {
            padding: 20px;
        }
        .btn {
            display: inline-block;
            background-color: #28a745;
            color: white;
            padding: 10px 20px;
            text-decoration: none;
            border-radius: 5px;
            font-size: 18px;
        }
        .btn:hover {
            background-color: #218838;
        }
        footer {
            background-color: #343a40;
            color: white;
            padding: 10px;
            position: fixed;
            width: 100%;
            bottom: 0;
        }
    </style>
</head>
<body>

    <header>
        جمعية الخير والعطاء
    </header>

    <section>
        <h2>مهمتنا</h2>
        <p>نحن نساعد المحتاجين من خلال التبرعات والمبادرات الخيرية لنشر الخير في المجتمع.</p>
        <a href="#" class="btn">تبرع الآن</a>
    </section>

    <footer>
        &copy; 2025 جمعية الخير والعطاء | جميع الحقوق محفوظة
    </footer>

</body>
</html>

**Review by mohammedijas80755-cell — COMMENTED** [review]
H2h2j

File: git-guides/git-add.md [hunk 3]
```diff
@@ -75,7 +75,7 @@ If the time is right to stage all files, there are several commands that you can
 * `git add .`: adds the entire directory recursively, including files whose names begin with a dot
 * `git add -u`: stages modified and deleted files only, NOT new files
 
-| | New files | Modified files | Deleted files | Files with names beginning with a dot | Current directory | Higher directories|
+| Command | New files | Modified files | Deleted files | Files with names beginning with a dot | Current directory | Higher directories|
```
Comment (mohammedijas80755-cell) [inline git-guides/git-add.md]: Ywuw

File: git-guides/git-add.md [hunk 4]
```diff
@@ -75,7 +75,7 @@ If the time is right to stage all files, there are several commands that you can
 * `git add .`: adds the entire directory recursively, including files whose names begin with a dot
 * `git add -u`: stages modified and deleted files only, NOT new files
 
-| | New files | Modified files | Deleted files | Files with names beginning with a dot | Current directory | Higher directories|
```
Comment (mohammedijas80755-cell) [inline git-guides/git-add.md]: Gagwhw

**Pull request merged**